documentation/aircraft-commands.md

```diff
@@ -319,27 +329,32 @@ _Syntax -_ `AAL123 fph`
 
 _Aliases -_ `heading` / `h` / `turn` / `t` / `fh`
 
-_Shortkeys -_ `left arrow` / `right arrow` (if "Control Method" setting = "Arrow Keys")
+_Hotkeys -_ `left arrow` / `right arrow` (if "Control Method" setting = "Arrow Keys")
 
-_Information -_ This command sets the target heading; up (north) is 360,
-right (east) is 090, down (south) is 180, and left (west) is 270. Of course
-you can use any angle in between these as well. If the heading is set
-before takeoff, the aircraft will turn to that heading after takeoff. You
-can force the aircraft to reach the heading by turning left or right by
-inserting `l` or `r` before the new heading, as demonstrated below.
+_Information -_ This command tells the aircraft the target heading to fly
+towards: up (north) is 360, right (east) is 090, down (south) is 180, and left
+(west) is 270. Of course you can use any angle in between these as well. If the
+heading is set before takeoff, the aircraft will turn to that heading after takeoff.
 
-_Syntax -_ `AAL123 fh [hdg]` or `AAL123 (rightarrow) [hdg]` or `AAL123 t r [hdg]`
+The target heading is specified by a three-digit heading. You can force the
+aircraft to reach the heading by turning left or right by inserting `left`,
+`l`, `right`, or `r` before the new heading, as demonstrated below.
 
-### Speed
+Alternatively, you can instruct an aircraft to turn from its current heading by
+a given angle by specifying the direction of turn with the heading change as
+a one- or two-digit number of degrees
 
-_Aliases -_ `speed` / `slow` / `sp`
+_Syntax -_ `AAL123 fh [hdg]` or `AAL123 (rightarrow) [hdg]` or `AAL123 t r 270` (target heading) or `AAL123 t r 45` (heading change)
+
+### Speed
 
-_ShortKey -_ `numpad +` / `numpad -`
+_Aliases -_ `speed` / `slow` / `sp` / `+` / `-`
 
 _Information -_ This command sets the target speed; aircraft will stay within
 their safe speeds if you tell them to fly faster or slower than they are able
 to. It takes some time to increase and reduce speed. Remember that speed is
-always expressed in knots.
+always expressed in knots, and is indicated airspeed, whereas our radar scope
+can only display groundspeed.
```

Comment:
Good change. In fact, knots is super obvious (and beside the point), so let's simplify to:
`Remember that speed assignments are given in indicated airspeed, whereas our scope only displays groundspeeds.`

Reply:
done